graphitron-codegen-parent/graphitron-java-codegen/src/test/java/no/sikt/graphitron/queries/fetch/ReferenceInputTest.java

```diff
@@ -149,4 +149,16 @@ void onSingleTableInterface() {
                 ".leftJoin(_a_address_223244161_city_left)"
         );
     }
+
+    @Test
+    @DisplayName("Input with table and return type with field having reference")
+    void inputWithTableReturnTypeWithReference() {
+        var generated = generateFiles("inputWithTableReturnTypeWithReference");
```

Comment:
```suggestion
        var generated = generateFiles("inputWithTableReturnTypeWithScalarReference");
```

Så vidt jeg kan se gjelder bare dette skalarer